src/Dialect/ONNX/Rewrite.cpp

```diff
@@ -179,6 +179,45 @@ bool AreTheSameAxesConstant(int64_t rank, Value lhs, Value rhs) {
 
 #include "src/Dialect/ONNX/ONNXRewrite.inc"
 
+// =============================================================================
+// Rewrite pattern for Resize (not handled in Rewrite.td).
+// =============================================================================
+
+// The yolo4 model uses a float tensor with shape [0] to represent that roi
+// or scales is absent (in violation of the spec which says that empty string
+// inputs represents absent arguments in the protobuf model representation).
+// We work around this by interpreting a tensor with empty shape as an
+// alternative way to express that an input is absent.
+class EmptyTensorInputsResizePattern : public OpRewritePattern<ONNXResizeOp> {
```

Comment:
Could you please add lit tests for this rewriting? Thanks!

Reply:
good suggestion

done